File: 1. Two Sum/Two Sum.cs
public class Solution
{
	public int[] TwoSum(int[] nums, int target)
	{
		var m = new Dictionary<int, int>();
		for (var i = 0; i < nums.Length; ++i)
		{
			var x = target - nums[i];
			if (m.ContainsKey(x))
			{
				return new[] { m[x], i };
			}
			m[nums[i]] = i;
		}
		throw new ArgumentException(@"找不到");
	}
}
//Runtime: 280 ms

File: 1. Two Sum/Two Sum.kt
class Solution
{
	fun twoSum(nums:IntArray, target:Int):IntArray
	{
		var m = mutableMapOf<Int, Int>()
		for(i in 0 until nums.size)
		{
			var x=target-nums[i]
			if(m.containsKey(x))
			{
				return intArrayOf(m.getValue(x),i)
			}
			m[nums[i]]=i
		}
		throw IllegalArgumentException("""没找到""")
	}
}
//Runtime: 220 ms